README.md

```diff
@@ -23,32 +23,7 @@ installed. This helps with making the builds reproducible and the provenances
 verifiable. The toml file should conform to the `BuildConfig` structure defined
 in the [`common`](/common/) package.
 
-The [`cmd/build`](/cmd/build/) command line tool described above can be used
```

Comment:
I think this paragraph is important and should be included. This describes how to provide a config file similar to `testdata/build.toml`.

Reply:
Added it back in---sorry!

Reply:
Thanks :)